docs/features/metrics.md

```diff
@@ -96,7 +93,84 @@ distinct-registrations:
     where user_id is not null
 ```
 
-\`\`\`
```

Comment:
I think these three correspond to this in the documentation. I guess we can remove it?
![image](https://user-images.githubusercontent.com/52512373/101230996-9a6fef80-3676-11eb-9545-9e46a61c5deb.png)